modules/support-collecting-host-network-trace.adoc

```diff
@@ -51,9 +59,36 @@ $ oc adm must-gather \
 <.> The `--host-network=true` argument is required so that the packet captures are performed on the network interfaces of the node.
 <.> The `--timeout` argument and value specify to run the debug pod for 30 seconds. If you do not specify the `--timeout` argument and a duration, the debug pod runs for 10 minutes.
 <.> The `-i any` argument for the `tcpdump` command specifies to capture packets on all network interfaces. As an alternative, you can specify a network interface name.
+endif::openshift-origin[]
+
```

Comment:
@peer-review-squad Existing text in the following edit. I updated only the image name in the code block and added ifdef